examples/rotator_test_light.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# coding: utf-8
+
```

Comment:
Need a top level description at the top what this script is doing and what the goal is.

Reply:
If this is an example of something, you should also state what you think this could be the base of (e.g. a different type of rotator test? or expanded to any CSC?)